src/index.ts

```diff
@@ -33,6 +33,18 @@ export default defineWebApplication({
           {
             extension: 'glb',
             label: 'View 3D Model'
+          },
+          {
+            extension: 'stl',
+            label: 'View STL 3D Model'
+          },
+          {
+            extension: 'fbx',
+            label: 'View FBX 3D Model'
+          },
+          {
+            extension: 'obj',
+            label: 'View OBJ 3D Model'
```

Comment:
general label should be fine.
```suggestion
            label: 'View 3D Model'
          },
          {
            extension: 'fbx',
            label: 'View 3D Model'
          },
          {
            extension: 'obj',
            label: 'View 3D Model'
```